app/src/main/java/com/goayo/debtify/model/DebtTracker.java

```diff
@@ -79,6 +79,15 @@ public double getSumOfPayments() {
         return sum;
     }
 
+    /**
+     * Calculates remaining debt.
+     *
+     * @return the remaining amount to be payed.
+     */
+    public double getRemainingDebt() {
+        return debt.getDebtAmount() - getSumOfPayments();
```

Comment:
this method exists already and is called getAmountOwed()

Reply:
Switched over to the existing method!